.github/workflows/workflow.yml

```diff
@@ -47,6 +48,12 @@ jobs:
       - run: yarn --frozen-lockfile
         working-directory: cohttp-lwt-jsoo/test
 
+      - run: |
+          sudo apt update
+          sudo apt upgrade
```

Comment:
```suggestion
```

I just made one more test, it works also without this step (and it is slightly faster)

Reply:
I am testing if we can also drop the next one

Reply:
@mseri could we do these small tweaks on the main branch itself after this PR is merged?

Reply:
Sure. I can probably also push in this branch myself. I am currently waiting a bit longer to see if I can get extra feedback